GCTC-NTGC/gc-digital-talent
# Copy - 3905 - IAP Launch Copy

Resolves #3905 

## Summary

This updates the copy for the "Launch" section to be "later half of 2022" in both English and French.

## Screenshots

### English

<img width="447" alt="Screen Shot 2022-09-09 at 2 53 25 PM" src="https://user-images.githubusercontent.com/4127998/189423553-30cad1ad-f208-4018-9204-48b4e5a5a239.png">

### French

<img width="383" alt="Screen Shot 2022-09-09 at 2 53 16 PM" src="https://user-images.githubusercontent.com/4127998/189423599-b5002955-374d-4e9b-b3c2-02bcf8ac60fe.png">

## Testing

1. Compile French copy `npm run intl-compile --workspace=indigenousapprenticeship`
2. Build the IAP project `npm run production --workspace=indigenousapprenticeship `
3. Confirm the launch section has new copy
